content/posts/2019/11/libplanet-0.7/index.eng.md

```diff
@@ -70,7 +70,7 @@ Currently, [key derivation function] implements [PBKDF2] and [Scrypt], and suppo
 Furthermore,
 ----
 
-This version includes many contributions from the [Hacktobberfest] event. You can learn more about the event in [Looking Back on Hacktoberfest] [Looking-back-at-hacktobberfest] written by Swen Mun, and all other changes made to this version can be found in [Full Changes] [0.7.0].
+This version includes many contributions from the [Hacktoberfest] event. You can learn more about the event in [Looking Back on Hacktoberfest][looking-back-at-hacktobberfest] written by Swen Mun, and all other changes made to this version can be found in [Full Changes] [0.7.0].
```

Comment:
Though this part won't appear in the article, I think it's better to change the part if you can.

```suggestion
This version includes many contributions from the [Hacktoberfest] event. You can learn more about the event in [Looking Back on Hacktoberfest][looking-back-at-hacktoberfest] written by Swen Mun, and all other changes made to this version can be found in [Full Changes] [0.7.0].
```